app/views/policies/_form.html.erb

```diff
@@ -5,7 +5,8 @@
   <%= base_errors_for @policy %>
 
   <ul class="nav nav-tabs" data-tabs="tabs">
-    <li class="active"><a href="#primary" data-toggle="tab"><%= _("General") %></a></li>
+    <li class="active"><a href="#deployment" data-toggle="tab"><%= _("Deployment Options") %></a></li>
```

Comment:
in wizard we call it Client Deployment, if we plan to add more options in future, let's use this but also in the wizard